architecture-decision-records/0021-service-unavailable-page.md

```diff
@@ -0,0 +1,37 @@
+# 20. Deploying a service unavailable page 
+
+**Date:** 2021-07-14
+
+## Context
+We need a mechanism to manually deploy a service unavailable page. There are two main reasons we might want to do this. 
+* There is a serious issue with the site that we can't fix immediately.
+* We want to carry out maintenance which will affect service. 
+
+The page will be a single static html page which will be displayed when a user goes to any of the https://tdr.nationalarchives.gov.uk/* urls.
+
+![Service Unavailable Design](images/service-unavailable-design.png)
+
+## Evaluation criteria
+* We must be able to deploy the page manually.
```

Comment:
Possibly add an additional criteria:

- that it should be easy for anyone on the TDR dev team + David C to deploy. 
- easy to switch back

Or something along those lines.

Reply:
Yeah, good point. I've added those two in. I've put TDR team as anyone with Jenkins access can deploy it which I think is everyone.